apps/ui/src/components/checkbox/styles/variables.css

```diff
@@ -0,0 +1,36 @@
+:root {
```

Comment:
I recommend using a prefix called `--checkbox-**` on **all checkbox css  variables** ex: `--checkbox-color-primary` so that it wont be a conflict with other css variables